cmd_scoreboard.lua

```diff
@@ -0,0 +1,234 @@
+
+	if Split[3] == "get" then
+		tObjective:GetScore(tplayer)
+		return true
+	end
+
+	if Split[3] == "set" then
+		if not Split[6] then return false end
+		tObjective:SetScore(tplayer, Split[6])
+		return true
+	end
+
+	if Split[3] == "add" then
+		if not Split[6] then return false end
+		tObjective:AddScore(tplayer, Split[6])
+		return true
+	end
+
+	if Split[3] == "remove" then
+		if not Split[6] then return false end
+		tObjective:SubScore(tplayer, Split[6])
+		return true
+	end
+
+	if Split[3] == "operation" then
+		if not Split[8] then return false end
+		local operation = Split[6]
+		local sourcePlayer = Split[7]
+		local sourceObjective = Scoreboard:getObjective(Split[8])
+
+		if operator == "+=" then
+			tObjective:SetScore(tPlayer, tObjective:GetScore(tPlayer) + sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "-=" then
+			tObjective:SetScore(tPlayer, tObjective:GetScore(tPlayer) - sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "*=" then
+			tObjective:SetScore(tPlayer, tObjective:GetScore(tPlayer) * sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "/=" then
+			tObjective:SetScore(tPlayer, tObjective:GetScore(tPlayer) / sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "%=" then
+			tObjective:SetScore(tPlayer, tObjective:GetScore(tPlayer) % sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "=" then
+			tObjective:SetScore(tPlayer, sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+		if operator == "<" then
+			if sourceObjective:GetScore(sourcePlayer) < tObjective:GetScore(tPlayer) then
+				tObjective:SetScore(tPlayer, sourceObjective:GetScore(sourcePlayer))
+			end
+			return true
+		end
+		if operator == ">" then
+			if sourceObjective:GetScore(sourcePlayer) > tObjective:GetScore(tPlayer) then
+				tObjective:SetScore(tPlayer, sourceObjective:GetScore(sourcePlayer))
+			end
+			return true
+		end
+		if operator == "><" then
+			local temp = sourceObjective:GetScore(sourcePlayer)
+			sourceObjective:SetScore(tObjective:GetScore(tPlayer))
+			tObjective:SetScore(tPlayer, sourceObjective:GetScore(sourcePlayer))
+			return true
+		end
+	end
+	return false
+end
+
+function listObjectiveofPlayer(Objective)
+	local score = Objective:GetScore(tplayer)
+	if score then
+		if gPlayer then
+			gPlayer:SendMessageInfo(Objective:GetDisplayName() .. " -> " .. score)
+		else
+			LOG(Objective:GetDisplayName() .. " -> " .. score)  -- TODO
+		end
+	end
+end
+
+function resetAllObjectives(Objective)
+	Objetive:ResetScore(tplayer)
+end
+
+function sendListObjectives(Objective)
+	if gPlayer then
+		gPlayer:SendMessage(Objective:GetDisplayName() .. " -> " .. Objective:GetName() .. ": " .. get_key_for_value(criterias, Objective:GetType()))
+	else
+		LOG(Objective:GetDisplayName() .. " -> " .. Objective:GetName() .. ": " .. get_key_for_value(criterias, Objective:GetType()))  -- TODO
```

Comment:
I was wondering... is there a better way to send the result of a command (with multiple line) than doing it with `LOG()` ?

Reply:
This is how the /tps command does it: https://github.com/cuberite/Core/blob/baf6a4e7050f8ed3c19dab50924863ed704a3eba/cmd_tps.lua#L34-L42

I don't remember if we have other commands in the new format that output multiple lines.